vespalib/src/tests/thread/thread_test.cpp

```diff
@@ -6,6 +6,8 @@
 
 using namespace vespalib;
 
+VESPA_THREAD_STACK_TAG(test_agent_executor);
```

Comment:
Consider _thread instead of _executor as this is used for vespalib::Thread.

Reply:
Fixed